config/200-clusterroles.yaml

```diff
@@ -277,6 +277,14 @@ rules: []
 
 ---
 
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: awsperformanceinsightssource-adapter
```

Comment:
Why again btw? It's the same as above.

Reply:
no idea how that happened, to be honest.